meta/boardreports/2022-11.md

```diff
@@ -0,0 +1,40 @@
+# Patterns Board Report (Nov)
```

Comment:
```suggestion
# Patterns Board Report - for Board Meeting 2022-11
```

Reply:
will update just updated that to add "2022" so won't let me commit your suggestion but makes sense.